app/models/loan.rb

```diff
@@ -136,6 +141,12 @@ def checked_out?
     ended_at.blank?
   end
 
+  def renewal_request
```

Comment:
I think the name of this method could be a little more specific. `unapproved_renewal_request` or something that is a little more explicit would make the intent a bit clearer.